corehq/pillows/xform.py

```diff
@@ -97,7 +97,7 @@ def transform_xform_for_elasticsearch(doc_dict, include_props=True):
         except KeyError:
             user_id = None
         doc_ret['user_type'] = get_user_type(user_id)
-        doc_ret['inserted_at'] = datetime.datetime.now(pytz.utc).isoformat()
+        doc_ret['inserted_at'] = datetime.datetime.utcnow().isoformat()
```

Comment:
we store timezone naive dates and our ES config doesn't allow us to submit timezone aware dates which is why the reindex kept failing. see: https://github.com/dimagi/commcare-hq/blob/master/corehq/pillows/core.py#L3. there's no format for the `+00:00` timezone. odd that it works locally, but think that's a configuration thing